### ISSUE TITLE
Order on column of included table

### ISSUE THREAD

**Issue by SirTrollsalot** [mevdschee/php-crud-api#331]
Is it possible to order a List by an includet Column?

for example:
api.php/tbl_contacts?columns=contact_id,Name,EMail,tbl_contacttype.Description&include=tbl_contacttype&order=tbl_contacttype.Description&page=1,30

**Comment by mevdschee** [comment]
no, that is not possible

**Comment by SirTrollsalot** [comment]
Then i guess i also can't filter on those columns

**Comment by mevdschee** [comment]
Yes or you can do two request, where you retrieve id's in the first and then do an "in" filter in the second. You need to combine those two results (manually) in JavaScript. Good luck!

**Comment by mevdschee** [comment]
I'll keep this open as a feature request.

**Comment by SirTrollsalot** [comment]
Hey it's me.... again...i'm sorry...
Is there any way i can combine All and Any in the Filter?
so logically:
Filter[]=id,cs,1  AND (Filter[]=name,cs,text OR Filter[]=adress,cs,text)

I do not want to download it and combine it manually, because i am using pages.
Downloading everything would also be bad because of performance reasons

**Comment by mevdschee** [comment]
>Hey it's me.... again...i'm sorry...

Sure.. no problem. Although different issues would be better in different tickets :-).

> Is there any way i can combine All and Any in the Filter?

This currently not supported as there is no nesting syntax.

**Comment by mevdschee** [comment]
You asked:

> Is there any way i can combine All and Any in the Filter?

I answered:

>This currently not supported as there is no nesting syntax.

But this is no longer true (in v2), see: 

https://github.com/mevdschee/php-crud-api#multiple-filters

You can now combine AND and OR :-)

Enjoy!

**Comment by mevdschee** [comment]
There are no plans to support sorting on fields of included tables.

**Comment by mevdschee** [comment]
I'll keep this open as a feature request.

**Comment by mevdschee** [comment]
filtering on included tables is now supported.
